src/application/trading-copilot/commands/remove-trading-copilot-subscription.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Command,
+  FailureResult,
+  HandlerError,
+  OkResult,
+} from 'shared/messaging';
+
+import { Subscription } from '../types';
+import { TradingCopilotSettingsManager } from '../subscriptions-manager';
+
+interface Payload {
+  ensName: Subscription['ensName'];
+}
```

Comment:
Feel free to use types already :D we should migrate fully to types soon... whenever we're not having big PRin progress, btw. no need to change it now, just mentioning